phpBB/viewforum.php

```diff
@@ -710,10 +710,28 @@
 		'SELECT'		=> $sql_array['SELECT'],
 		'FROM'			=> $sql_array['FROM'],
 		'LEFT_JOIN'		=> $sql_array['LEFT_JOIN'],
-
 		'WHERE'			=> $db->sql_in_set('t.topic_id', $topic_list),
 	);
 
+	/**
+	* Event to modify the SQL query before obtaining topics/stickies
+	*
+	* @event core.viewforum_modify_topics_sql
+	* @var	int		forum_id			The forum ID
+	* @var	array	forum_data			Data about the forum
+	* @var	array	topic_list			Topic ids array
+	* @var	array	sql_array			SQL query array for obtaining topics/stickies
```

Comment:
I would prefer `@var array` with a single space, rather than a tab.

Reply:
Ok. 
But what about this event : https://github.com/dark-1/phpbb/blob/ticket/16381/phpBB/phpbb/session.php#L1090
there it's used tabs & it's `@since 3.2.9-RC1` ???

Reply:
I think I'll keep it as per `coding-guidelines` , once your discussion topic comes on a final decision, and the `coding-guidelines` are updated , I'll follow that henceforth.